system/help/help.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * Toggles the visibility of the information icons.
+ *
+ * @returns void
+ */
+function toggleInfoButtons(e) {
+	$(e).toggleClass('clicked');
+	$('.hyphaInfoButton').toggle();
+}
+
+/**
+ * Creates an info popup.
+ *
+ * @param anchor Element, mouse click event, used to determine position of popup
+ * @param key string, the key for with to get the info
+ * @param language string
+ * @param info Element, the info button for which to make an info popup
+ */
+function toggleInfoPopup(anchor, key, language, info) {
+	// If it has a clicked state, close the popup and remove the state.
+	var $info = $(info);
+	if ($info.hasClass('clicked')) {
+		$info.trigger('closeMe');
+		return;
+	}
+
+	// Directly set the clicked class, this also prevents the spamming of the system before the result is returned.
+	$info.addClass('clicked');
+
+	// Try to get page language from selected langlist.
+	if (!language) {
+		var $selectedLanguage = $('#langList .selected');
+		if ($selectedLanguage.length) {
+			language = $selectedLanguage[0].innerHTML;
```

Comment:
This doesn't look entirely clean to me. Why check the page language at all and not just stick to the interface language?